src/test/java/de/caritas/cob/consultingtypeservice/api/controller/TopicControllerIT.java

```diff
@@ -69,7 +70,8 @@ void getTopicList_Should_ReturnTopicsList_When_UserIsAuthenticated()
     AuthenticationMockBuilder builder = new AuthenticationMockBuilder();
     mockMvc.perform(
             get(TopicPathConstants.PATH_GET_TOPIC_LIST)
-                .with(authentication(builder.withAuthority(UserRole.TOPIC_ADMIN.getValue()).build()))
+                .with(
+                    authentication(builder.withAuthority(UserRole.TOPIC_ADMIN.getValue()).build()))
```

Comment:
I think I don't get this. change the test? Or what do you mean?

Reply:
no, I mean the code itself, since you introduced publicly available endpoint to get all active topics, we need to enforce that only the topic-admin can get to the getAll endpoints (currently it's not the case, please see the Controller class and look for @PreAuthorize annotations)

Reply:
got it

Reply:
yes

Reply:
if this is still so, I can change this.